docs/event-schemas/workflowmanager/WorkflowRunStateChange.json

```diff
@@ -54,8 +54,9 @@
           "portalRunId",
           "timestamp",
           "status",
```

Comment:
I wonder if we should rename this to "state"... 
It would align better with the "state change" events.

Reply:
ignore!

Reply:
We discussed together. Conclusion was - state change might constitute multiple combination of different property changes - e.g. `combo(timestamp, status, portalRunId)`